signer/src/transaction_coordinator.rs

```diff
@@ -390,6 +392,95 @@ where
         .map(|_| ())
     }
 
+    /// Constructs a BitcoinPreSignRequest from the given transaction package and
+    /// sends it to the signers. Waits for acknowledgments from the signers until
+    /// the threshold is met or a timeout occurs.
+    /// If the signal stream closes unexpectedly, triggers a shutdown.
+    #[tracing::instrument(skip_all)]
+    async fn construct_and_send_bitcoin_presign_request(
+        &mut self,
+        bitcoin_chain_tip: &model::BitcoinBlockHash,
+        pending_requests: &SbtcRequests,
```

Comment:
nit: we only need the `SignerBtcState` here. Also, it might be a little confusing using this type with the `sbtc_requests` variable below.

Reply:
done in https://github.com/stacks-network/sbtc/pull/956/commits/c803d9d18bc48fc3aae449ca3c59233a26f44228